CI-CD/Jenkins/Jenkins_freestyle.sh

```diff
@@ -0,0 +1,10 @@
+echo "Downloading WS"
+if ! [ -f ./wss-unified-agent.jar ]; then
+  curl -fSL -R -JO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+fi
+echo "Exceute WS"
+export WS_APIKEY=${APIKEY} #Taken from Jenkins Global Environment Variables
```

Comment:
Add WS_USERKEY as this is a best practice

Reply:
added